css/style.css

```diff
@@ -4629,259 +4627,1974 @@ input[type="button"].btn-block {
   font-size: 1.5em;
   font-weight: 700;
   line-height: 1;
```

Comment:
@lynnnakiganda , there is extra space in this file. Between line 4630 and 6345. Please remove it.